packit.yml

```diff
@@ -0,0 +1,29 @@
+---
+# https://packit.dev/docs/configuration/
+
+specfile_path: simple-term.spec
+files_to_sync: [simple-term.spec, packit.yml]
+upstream_package_name: simple-term
+downstream_package_name: simple-term
+
+jobs:
+  - job: copr_build
+    trigger: pull_request
+    targets: [fedora-all]
+
+  - job: tests
+    trigger: pull_request
+    targets: [fedora-all]
```

Comment:
This is overkill -- simple-term doesn't have any FMF tests, so the only thing that this will do is to check that the rpms install (which isn't a surprise, there are no %post scripts, services, anything)

Reply:
https://github.com/mh21/simple-term/pull/11